pkg/certs/cert-inspection/certgraphanalysis/analyzer.go

```diff
@@ -77,6 +85,92 @@ func InspectConfigMap(obj *corev1.ConfigMap) (*certgraphapi.CertificateAuthority
 		return nil, err
 	}
 	caBundleDetail = addConfigMapLocation(caBundleDetail, obj.Namespace, obj.Name)
+	return caBundleDetail, nil
+}
+
+func extractKubeConfigFromConfigMap(obj *corev1.ConfigMap) (*rest.Config, error) {
+	if obj == nil {
+		return nil, fmt.Errorf("empty object")
+	}
+	for _, v := range obj.Data {
```

Comment:
this appears to be parsing every key of potentially every configmap to see if it's a kubeconfig?  Instead of doing this, could we choose specific keys we find valid?

Reply:
Yeah, I can modify this to a hardcoded list of keys, but I'm worried that it might miss some new kubeconfigs.